pkg/volume/vspherevolume/vsphere_volume_util.go

```diff
@@ -36,37 +36,35 @@ import (
 )
 
 const (
-	maxRetries         = 10
-	checkSleepDuration = time.Second
-	diskByIDPath       = "/dev/disk/by-id/"
-	diskSCSIPrefix     = "wwn-0x"
-	diskformat         = "diskformat"
-	datastore          = "datastore"
-	StoragePolicyName  = "storagepolicyname"
-
-	HostFailuresToTolerateCapability    = "hostfailurestotolerate"
-	ForceProvisioningCapability         = "forceprovisioning"
-	CacheReservationCapability          = "cachereservation"
-	DiskStripesCapability               = "diskstripes"
-	ObjectSpaceReservationCapability    = "objectspacereservation"
-	IopsLimitCapability                 = "iopslimit"
-	HostFailuresToTolerateCapabilityMin = 0
-	HostFailuresToTolerateCapabilityMax = 3
-	ForceProvisioningCapabilityMin      = 0
-	ForceProvisioningCapabilityMax      = 1
-	CacheReservationCapabilityMin       = 0
-	CacheReservationCapabilityMax       = 100
-	DiskStripesCapabilityMin            = 1
-	DiskStripesCapabilityMax            = 12
-	ObjectSpaceReservationCapabilityMin = 0
-	ObjectSpaceReservationCapabilityMax = 100
-	IopsLimitCapabilityMin              = 0
+	checkSleepDuration                  = time.Second
```

Comment:
Any reason why you unexported these constants? Are these never used somewhere in k/k or somewhere else?

Reply:
Yes i think so.